src/app/api/auth/register/route.ts

```diff
@@ -0,0 +1,48 @@
+import { NextRequest, NextResponse } from "next/server";
+import { z } from "zod";
+import { createUserWithEmailAndPassword, updateProfile } from "firebase/auth";
+import { auth } from "@/config/firebase";
+import { createUser } from "@/lib/db";
+import { sendWelcomeEmail } from "@/lib/sendWelcomeEmail";
+import { SignUpSchema } from "@/app/utils/validate/SignUpSchema";
```

Comment:
```suggestion
import { auth } from "@config/firebase";
import { createUser } from "@lib/db";
import { sendWelcomeEmail } from "@lib/sendWelcomeEmail";
import { SignUpSchema } from "@app/utils/validate/SignUpSchema";
```

it should be without the `@/`